http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200320_is_updating.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="azmina" id="is_updating">
+        <addColumn tableName="CLUSTER">
```

Comment:
Could this be:
```
<addColumn tableName="CLUSTER">
   <column name="isUpdating" type="boolean" defaultValue="false">
      <constraints nullable="false" />
   </column>
 </addColumn>
```
Then we don't need the 2 changeSets below?

Reply:
Would that take care of previous clusters that don't have this column? I was under the impression that the default value was for future rows of the database

Reply:
I think adding a default populates all existing null values… haven't tested with liquibase though.

Reply:
I don't think the field should be added here..it should be in `followupDetails` table

Reply:
I thought we wanted it on the CLUSTER table? It specifies whether a cluster has an update in progress, and is returned by listClusters/getCluster.